src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManager.php

```diff
@@ -447,6 +447,15 @@ public function save($uploadedFile, $data, $userId)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function saveEntity(\Sulu\Bundle\MediaBundle\Entity\Media $media)
+    {
+        $this->em->persist($media);
+        $this->em->flush();
+    }
```

Comment:
I would not introduce this function... Also, where you are using it, you don't need the call to persist, so I would just call flush there. (Although I am not sure if that you need something like this is the best idea either...) Isn't there a possibility that you can use one of the existing methods.